domain_specific.tex

```diff
@@ -0,0 +1,78 @@
+dyadic objects that allow for three dimensional operations such as addition,
+subtraction, scalar multiplication, inner and outer products, cross products,
+etc. Both of these objects can be written in very compact notation that make it
+easy to express the vectors and dyadics in terms of multiple reference frames
+with arbitrarily defined relative orientations. The vectors are used to specify
+the positions, velocities, and accelerations of points, orientations, angular
+velocities, and angular accelerations of reference frames, and force and
+torques. The dyadics are essentially reference frame aware 3 x 3 tensors. The
+vector and dyadic objects can be used for any one, two, or three dimensional
+vector algebra and provide a strong framework to build physics and engineering
+tools atop.
+
+\begin{listing}
+  \begin{minted}{pycon}
+>>> A = ReferenceFrame('A')
+>>> B = ReferenceFrame('B')
+>>> C = ReferenceFrame('C')
+>>> B.orient(A, 'body', (1, 2, 3), 'xzx')
+>>> C.orient(B, 'axis', (4, B.x))
+>>> v = 1 * A.x + 2 * B.z + 3 * C.y
+>>> v
+A.x + 2*B.z + 3*C.y
+>>> v.express(A)
+(-3*sin(2)*cos(3)*cos(4) + 3*sin(2)*sin(3)*sin(4) + 2*sin(2)*sin(3) + 1)*A.x + (3*(-sin(3)*cos(1)*cos(2) - sin(1)*cos(3))*sin(4) + 3*(-sin(1)*sin(3) + cos(1)*cos(2)*cos(3))*cos(4) - 2*sin(3)*cos(1)*cos(2) - 2*sin(1)*cos(3))*A.y + (2*cos(1)*cos(3) + 3*(sin(3)*cos(1) + sin(1)*cos(2)*cos(3))*cos(4) - 2*sin(1)*sin(3)*cos(2) + 3*(cos(1)*cos(3) - sin(1)*sin(3)*cos(2))*sin(4))*A.z
+  \end{minted}
+  \caption{Python interpreter session showing how a vector is created using the
+    orthogonal unit vectors of three reference frames that are oriented with
+    respect to each other and the result of expressing the vector in one frame.}
+  \label{lis:physics-vector}
+\end{listing}
+
+\subsection{Classical Mechanics}
+
+The \verb|physics.mechanics| package utilizes the \verb|physics.vector| package
+to populate time aware particle and rigid body objects to fully describe the
+kinematics and kinetics of a rigid multi-body system. These objects store all
+of the information needed to derive the ordinary differential or differential
+algrebraic equations that govern the motion of the system, i.e. the equations
+of motion. These equations of motion abide by Newton's laws of motion and can
+handle any arbitrary kinematical constraints or complex loads. The package
+offers two automated methods for formulating the equations of motion based on
+Lagrangian Dynamics~\cite{Lagrange1811} and Kane's Method~\cite{Kane1985}. Lastly, there
+are autmated linearization routines for constrained dynamical
+systems based on ~\cite{Peterson2014}.
+
+\subsection{Quantum Mechanics}
+
+The \verb|sympy.physics.quantum| package provides quantum functions, states,
+operators, and computation of standard quantum models.
+
+% TODO : This needs some help from someone that knows something about quantum
+% physics. I wasn't able to understand much from the documentation.
```

Comment:
Let's just open an issue. Maybe I can convince @certik or @ellisonbg to write a few sentences.